<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomVacation</class>
 <widget class="QWidget" name="CustomVacation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plan Your Trip</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Plan Your Trip!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFirst">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSecond">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetFirst">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetSecond">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Teams Available</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination Teams (check preferred start team)</string>
   </property>
  </widget>
  <widget class="QListWidget" name="souvAvailable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Souvenir Available</string>
   </property>
  </widget>
  <widget class="QListWidget" name="souvSelected">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSouv1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSouv2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Souvenir added to your cart</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>411</width>
     <height>351</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>351</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>401</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>381</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="allTeamPlan">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start at New England Patriots</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bookYourTrip">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>750</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Book your Trip</string>
   </property>
  </widget>
  <widget class="QLabel" name="hiddenLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>570</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="result_lbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>770</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Perform DFS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Perform BFS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Spanning Tree</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="offer">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>60</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Offer Shortest Distance</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
